src/System.Net.Requests/src/System/Net/HttpWebResponse.cs

```diff
@@ -144,6 +144,12 @@ public virtual CookieCollection Cookies
                 CheckDisposed();
                 return _cookies;
             }
+
+            set
+            {
+                CheckDisposed();
+                _cookies = value;
+            }
```

Comment:
@stephentoub I thought the get covered it , but it did not , modified the get to set first and check.